Gemfile.lock

```diff
@@ -55,7 +55,7 @@ GEM
     rexml (3.2.6)
     rouge (4.1.2)
     safe_yaml (1.0.5)
-    sass-embedded (1.64.1-x86_64-linux-gnu)
+    sass-embedded (1.63.6-x86_64-linux-gnu)
```

Comment:
This looks like a downgrade. Probably just because you created this branch before I did the more recent `bundle update`.